usecases/objects/batch_references_add.go

```diff
@@ -117,6 +127,111 @@ func (b *BatchManager) validateReference(wg *sync.WaitGroup, ref *models.BatchRe
 	}
 }
 
+func validateReferenceMultiTenancy(ctx context.Context,
+	principal *models.Principal, schemaManager schemaManager,
+	repo VectorRepo, source *crossref.RefSource, target *crossref.Ref,
+	tenantKey string,
+) error {
+	if source == nil || target == nil {
+		return fmt.Errorf("can't validate multi-tenancy for nil refs")
+	}
+
+	sourceClass, targetClass, err := getReferenceClasses(
+		ctx, principal, schemaManager, source.Class.String(), target.Class)
+	if err != nil {
+		return err
+	}
+
+	// if both classes have MT enabled but different tenant keys,
+	// no cross-tenant references can be made
+	if sourceClass.MultiTenancyConfig != nil && targetClass.MultiTenancyConfig != nil {
```

Comment:
`MultiTenancyConfig` can be present, but still disabled (`.Enabled = false`)

Reply:
Fixed